yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -438,4 +438,18 @@ func NewLODatatypeIssue(objectType string, objectName string, SqlStatement strin
 	issue := loDatatypeIssue
 	issue.TypeName = fmt.Sprintf("%s on column - %s", issue.TypeName, colName)
 	return newQueryIssue(issue, objectType, objectName, SqlStatement, map[string]interface{}{})
-}
\ No newline at end of file
+}
+
+var multiRangeDatatypeIssue = issue.Issue{
+	Type:       MULTI_RANGE_DATATYPE,
+	TypeName:   "Unsupported datatype - multirange",
+	Suggestion: "Multirange data type is not yet supported in YugabyteDB, no workaround available currently",
+	GH:         "", //TODO
+	DocsLink:   "", //TODO
+}
+
+func NewMultiRangeDatatypeIssue(objectType string, objectName string, sqlStatement string, typeName string, colName string) QueryIssue {
+	issue := multiRangeDatatypeIssue
+	issue.TypeName = fmt.Sprintf("%s on column - %s", issue.TypeName, colName)
```

Comment:
Add the respective type
issue.TypeName = fmt.Sprintf("%s - %s on column - %s", issue.TypeName, typeName, colName)

Reply:
Done